pyaedt/modules/AdvancedPostProcessing.py

```diff
@@ -1111,6 +1105,8 @@ def evaluate_faces_quantity(
             Dictionary of parameters defined for the specific setup with values. The default is ``{}``.
         ref_temperature: str, optional
             Reference temperature to use for heat transfer coefficient computation. The default is ``""``.
+        time : str
```

Comment:
```suggestion
        time : str, optional
```

Reply:
You're right. I should've added `optional`. I'll add that in another PR.